plugin.video.mlbtv/addon.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<addon id="plugin.video.mlbtv" name="MLB.TV®" version="2023.3.28+matrix.1" provider-name="eracknaphobia">
+<addon id="plugin.video.mlbtv" name="MLB.TV®" version="2023.4.14+matrix.1+matrix.1" provider-name="eracknaphobia">
```

Comment:
The version number is not correct.

Reply:
sorry, i'm struggling with github actions and kodi addon submittor.

Reply:
Should be good now.